source/Irrlicht/CMeshSceneNodeInstanced.cpp

```diff
@@ -222,8 +222,10 @@ uint32_t CMeshSceneNodeInstanced::addInstance(const core::matrix4x3& relativeTra
 
 bool CMeshSceneNodeInstanced::addInstances(uint32_t* instanceIDs, const size_t& instanceCount, const core::matrix4x3* relativeTransforms, const void* extraData)
 {
-    uint32_t dummyBytes[instanceCount];
-    uint32_t aligns[instanceCount];
+    core::vector<uint32_t> dummyBytes_(instanceCount);
```

Comment:
good fix, however try to scope the vectors so they go out of scope after the `multi_alloc_addr`

Reply:
wanted to make it same scope as those arrays. Anyway `dummyBytes` is used once more after `multi_alloc_addr` (line 246)

Reply:
ok as you wish.